go/libraries/doltcore/table/table_writer.go

```diff
@@ -41,4 +41,5 @@ type TableWriteCloser interface {
 type SqlTableWriter interface {
 	TableWriteCloser
 	WriteSqlRow(ctx context.Context, r sql.Row) error
+	WriteSqlBatchedRow(ctx context.Context, r sql.Row) error
```

Comment:
Don't change this interface.

Rather, implement a new type of SqlTableWriter which has batched semantics